### PR TITLE
✋ humans.txt

### PR DESCRIPTION
**Pull request by mimecuvalo**
see: https://humanstxt.org/

@TodePond if you wanna help wordsmith/riff here, please do :) even after it lands

i've added this in a bunch of places :)
https://paper.dropbox.com/humans.txt
https://www.rewiringamerica.org/humans.txt
https://www.mobilize.us/humans.txt
https://www.joingreenwork.com/humans.txt
https://nightlight.rocks/humans.txt

### Change Type

- [x] `internal` — Any other changes that don't affect the published package[^2]

### CODE REVIEW

**Comment by SomeHats** [comment]
what if it just said "tldraw is gay"

**Comment by mimecuvalo** [comment]
> what if it just said "tldraw is gay"

"Our arrows are bendable, not necessarily straight" :P

File: apps/dotcom/public/humans.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+This site is made by possible by a plucky group of nerds in the London coding scene that are hacking together geometry, arrows, deli sandwiches, and huppiness.
+We aim to hack and craft our way through the unexplored map edges of the Sea of Coding.
+Also, it's a bit of Bret Victor fan club.
```
Comment (TodePond) [inline apps/dotcom/public/humans.txt:3]:
```suggestion
This website was coded entirely in Dreamberd.
```

File: apps/dotcom/public/humans.txt [hunk 2]
```diff
@@ -0,0 +1,49 @@
+This site is made by possible by a plucky group of nerds in the London coding scene that are hacking together geometry, arrows, deli sandwiches, and huppiness.
```
Comment (TodePond) [inline apps/dotcom/public/humans.txt:1]:
```suggestion
This very good whiteboard is made by possible by a plucky group of nerds in the London coding scene that are hacking together geometry, arrows, deli sandwiches, and huppiness.
```

**Pull request merged**